packages/client/hmi-client/src/components/model/petrinet/tera-parameter-other-value-modal.vue

```diff
@@ -142,17 +151,37 @@ import Column from 'primevue/column';
 import Row from 'primevue/row';
 import ColumnGroup from 'primevue/columngroup';
 import RadioButton from 'primevue/radiobutton';
+import TeraLineGraphic from './tera-line-graphic.vue';
 
 const props = defineProps<{
-	id: string;
+	id: any;
 	otherValueList: any;
 }>();
 
-const otherValueList = ref(props.otherValueList);
+const otherValueList = ref(
+	props.otherValueList.map((value, index) => {
+		value.index = index + 1;
+		return value;
+	})
```

Comment:
Generally speaking it is good practice to not use mutators in `map` functions.

Also would prefer to not change things coming in through `props`